packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -245,12 +249,12 @@ const helpers = {
   },
 
   negotiateOptionsForInvalidRequest(httpResponses: IHttpOperationResponse[]): IHttpNegotiationResult {
-    // currently only try to find a 400 response, but we may want to support other cases in the future
-    const code = '400';
-    const response = findResponseByStatusCode(httpResponses, code);
-    // TODO: what if no 400 response is defined?
+    // currently only try to find a 422 response, but we may want to support other cases in the future
+    const code = '422';
+    const response = findResponseByStatusCode(httpResponses, code, false);
+    // TODO: what if no 422 response is defined?
```

Comment:
Ok, so I know I said:

> the order is always: "we want to throw an X, if they have an X use it, if they have a default use it, they have neither so here is our version of an X"

but I think we should double this up for the validation use case because so many people use 400.... if validation fails can we look for X or Y?

> we want to throw an 422, if they have a 422 use it, if they have 400 that'll do, if they have a default use it, they have none of that so bugger here is our version of an 422

Is that bonkers?

Reply:
This makes sense, and it's happening in eb4cc18. Tests included.